servers/audio_server.h

```diff
@@ -56,15 +56,26 @@ class AudioDriver {
 	SafeNumeric<uint64_t> prof_time;
 #endif
 
+public:
+	struct SizePosition {
+		unsigned int size;
+		unsigned int position;
+
+		SizePosition(unsigned int p_size = 0, unsigned int p_position = 0) noexcept :
+				size(p_size), position(p_position) {}
+	};
```

Comment:
Created a whole class to fit it in one `std::atomic`, it's important that this values change together. Also it's just a 64-bit structure, so lock-free `std::atomic` will probably work on most devices (worked on my phone).
**EDIT**: It's actually not as important to change them together anymore, mostly for debugging purposes now
Copied from: https://github.com/godotengine/godot/pull/92969#discussion_r1634464829.